src/libpgmoneta/walfile/wal_reader.c

```diff
@@ -55,6 +61,16 @@ static bool is_included(char* rm, struct deque* rms,
                         uint64_t e_lsn, uint64_t end_lsn,
                         uint32_t xid, struct deque* xids, char** included_objects, char* rm_desc);
 
+static bool xlog_record_get_block_tag(struct decoded_xlog_record* record, int block_id,
+                                      struct rel_file_locator* rlocator, enum fork_number* frk,
+                                      block_number* blkno);
+
+static bool summarize_xlog_record(uint8_t info);
+static void summarize_xact_record_14(struct decoded_xlog_record* record, block_ref_table* brt);
```

Comment:
Maybe `summarize_xact_record_l15` and `summarize_xact_record_ge15`, just an idea